injector/src/main/java/edu/ucr/cs/riple/injector/Injector.java

```diff
@@ -147,14 +148,20 @@ public Set<FileOffsetStore> removeAnnotations(Set<RemoveAnnotation> requests) {
    */
   @Nullable
   public static CompilationUnit parse(Path path) {
-    // Set parser configuration to Java 17.
+    // TODO: Figure out how to differentiate a generated file from a non-generated file.
```

Comment:
We don't really need to differentiate a generated file from a non-generated file and this TODO might not be applicable here, would you please remove it. In both cases we would like to just return `null` and the caller should handle that.